<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImapTestWidget</class>
 <widget class="QWidget" name="ImapTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>131</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblServer">
       <property name="text">
        <string>Server</string>
       </property>
       <property name="buddy">
        <cstring>cmbServer</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbServer">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblPort">
       <property name="text">
        <string>Port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>spinPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinPort">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>143</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblLogin">
       <property name="text">
        <string>Login</string>
       </property>
       <property name="buddy">
        <cstring>cmbLogin</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbLogin">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblTls">
       <property name="text">
        <string>TLS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cmbTLS</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cmbTLS">
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Implicit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Explicit</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Password</string>
       </property>
       <property name="buddy">
        <cstring>lnPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lnPassword"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lblTimeout">
       <property name="text">
        <string>Timeout</string>
       </property>
       <property name="buddy">
        <cstring>spinTimeout</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinTimeout">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="5,2">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="chkMail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Check mailbox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbMessageFolder">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Inbox</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbMessageStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>All messages</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RECENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ANSWERED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UUANSWERED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FLAGGED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UNFLAGGED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SEEN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UNSEEN</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cmbMessageAlert">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Allert when size of all messages bigger than</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alert when mailbox contains more then</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alert when size of all messages bigger than</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
       <item>
        <widget class="QSpinBox" name="spinMessageSize">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>MB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
